project.clj

```diff
@@ -1,11 +1,7 @@
-(defproject ring/ring-accept "0.1.0-SNAPSHOT"
-  :description "FIXME: write description"
+(defproject jp.studist/ring-accept-language "0.1.0-SNAPSHOT"
+  :description "Ring-Accept-language provides a middleware that parses the Accept-language request header and injects the value into the request hash."
   :url "http://example.com/FIXME"
```

Comment:
You won't set an URL? studist.jp? biz.teachme.jp?

Reply:
according to reitit's project.clj, an url was set a Github url.
Therefore I've set Github url into an url key.

https://github.com/metosin/reitit/blob/8c9a7a42cb6ab9bb46535c604c3820019e1d88a1/project.clj#L3